Packs/SecLyticsAugurIOCsFeed/Integrations/SeclyticsFeed/SeclyticsFeed.py

```diff
@@ -0,0 +1,230 @@
+        """Builds a list of indicators.
+        Returns:
+            list. A list of JSON objects representing indicators fetched from a feed.
+        """
+        response = self.daily_http_request()
+        parsed_indicators = self.create_indicators_from_response(response, feed_tags,
+                                                                 tlp_color)  # list of dict of indicators
+
+        # for get_indicator_command only
+        if limit:
+            parsed_indicators = parsed_indicators[int(offset): int(offset) + int(limit)]
+        return parsed_indicators
+
+
+def module_test_command(client: Client, args: dict, feed_tags: list, tlp_color: Optional[str]):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client(Client): Seclytics Feed client
+        args(Dict): The instance parameters
+        feed_tags: The indicator tags
+        tlp_color: Traffic Light Protocol color
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    try:
+        client.build_iterator(argToList(demisto.params().get('feedTags')), tlp_color, 1000, 0)
+    except Exception:
+        raise Exception("Could not fetch Daily Threat Feed\n"
+                        "\nCheck your API key and your connection to Seclytics.")
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: dict, feed_tags: list, tlp_color: Optional[str]) \
+        -> Tuple[str, dict, list]:
+    """Initiate a single fetch-indicators
+
+    Args:
+        client(Client): The Seclytics Client.
+        args(dict): Command arguments.
+        feed_tags: The indicator tags.
+        tlp_color: Traffic Light Protocol color.
+    Returns:
+        str, dict, list. the markdown table, context JSON and list of indicators
+    """
+    offset = int(args.get('offset', 0))
+    limit = int(args.get('limit', 1000))
+
+    indicators = fetch_indicators_command(client, feed_tags, tlp_color, limit, offset)
+
+    hr_indicators = []
+    for indicator in indicators:
+        hr_indicators.append({
+            'Value': indicator.get('value'),
+            'Type': indicator.get('type'),
+            'rawJSON': indicator.get('rawJSON'),
+            'fields': indicator.get('fields'),
+        })
+
+    human_readable = tableToMarkdown("Indicators from Seclytics:", hr_indicators,
+                                     headers=['Value', 'Type', 'rawJSON', 'fields'], removeNull=True)
+
+    if args.get('limit'):
+        human_readable = human_readable + f"\nTo bring the next batch of indicators " \
+                                          f"run:\n!Seclytics-daily-get-indicators " \
+                                          f"limit={args.get('limit')} " \
+                                          f"offset={int(str(args.get('limit'))) + int(str(args.get('offset')))}"
+
+    return human_readable, {}, indicators
```

Comment:
in our new convention of using the `CommandResults` object 
you should return an instance of [CommandResults](https://xsoar.pan.dev/docs/integrations/code-conventions#commandresults)
you can see example of this implementation in return of [get_indicators_command](https://github.com/demisto/content/blob/ef5849eedec3d2e7cf0dd83d91f9eb32fe7cc9c3/Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py#L657)